src/Commands/Auth/WhoamiCommand.php

```diff
@@ -19,6 +25,15 @@ class WhoamiCommand extends TerminusCommand
      *   Responds with the current session and user's data
      */
     public function whoami() {
+        $auth = new Auth();
```

Comment:
It appears that $this->authorized is somehow always true, and login is being enforced before this runs.